cliconfig/file.go

```diff
@@ -82,10 +83,9 @@ func (f File) Exists() bool {
 //
 // The project is released under an MIT License, which can be seen here:
 // https://github.com/joho/godotenv/blob/master/LICENCE
-func parseLine(line string) (key string, value string, err error) {
```

Comment:
i've mostly changed this because i loathe named returns in go, but i'm happy to change things back if we want

Reply:
In my experience the main problem is bare returns. All your changes to the return statements are good!

Giving the return args names to aid documentation is usually a plus when there are multiple of the same type. 

But also `(key, value string, err error)` would have been slightly neater to begin with.

Reply:
oh yeah, bare returns are 90% of the problem. I might modify to get the best of both worlds